.github/workflows/ci.yaml

```diff
@@ -0,0 +1,94 @@
+name: PGContents CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  pre-commit:
+    name: pre-commit
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit-${{ hashFiles('.pre-commit-config.yaml') }}
+      - uses: pre-commit/action@v1.0.1
+
+  build-and-test:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        name:
+          - py27-notebook5
+          - py35-notebook5
+          - py35-notebook5
+          - py35-notebook6
+          - py36-notebook6
+        include:
+          - name: py27-notebook5
+            python-version: 2.7
+
+          - name: py35-notebook5
+            python-version: 3.5
+
+          - name: py36-notebook5
+            python-version: 3.6
+
+          - name: py35-notebook6
+            python-version: 3.5
+
+          - name: py36-notebook6
+            python-version: 3.6
+
+    services:
+      postgres:
+        image: "postgres:10-alpine"
```

Comment:
Was there a reason for bumping the postgres version?

Reply:
Nope. Setting this back to 9.5